stanford-earth/matson
# Earth-1088: Search Results and Search Styling

# READY

# Summary
- Add the search bar to the header, inside navigation area specifically
- Adjusted styles with buttons & opening the search bar
- Hide the menu links when the search bar is open
- Added search form to the top of search results page
- Styled search results to be in wide bordered rows
- moved all scss and js for search into the matson theme and OUT of css/js injectors

# Needed By (Date)
- End of week 5/31

# Urgency
- Highly requested feature

# Steps to Test

1. Pull down this branch
2. place 'Exposed form: acquia_search-page' block in Content region, above 'main page content', and display only on */search page
3. Update the search view: edit the Unformatted List > settings to have the class: complex_slat and we need to figure out how to update the view on the ode with all the field updates.. Katria will work on that with Dena.

3. Search page should show in the header on every page. It should open without a placeholder, and search correctly.
2. Menu links should gracefully hide when the header search bar is opened up.
3. On search results page, it should show the searched text in the input, and the results.
4. Different content types/user profiles have been formatted with several fields for the small red text label, the title, and the body/summary text

# Affected Projects or Products
- All work in the matson theme, only affects the Earth site

# Associated Issues and/or People
- JIRA tickets: 1088, 1094, 1371
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? @buttonwillowsix 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
